tests/io/test_cfradial2.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# Copyright (c) 2023-2025, openradar developers.
+# Distributed under the MIT License. See LICENSE for more info.
+
+import pytest
+import xarray as xr
+from open_radar_data import DATASETS
+
+import xradar as xd
+
+
+@pytest.fixture
+def cfradial1_file():
+    return DATASETS.fetch("cfrad.20080604_002217_000_SPOL_v36_SUR.nc")
+
+
+@pytest.fixture
+def temp_file(tmp_path):
+    return tmp_path / "cfradial2.nc"
+
+
+def test_open_cfradial2(temp_file, cfradial1_file):
+    # Open the original file using CfRadial1 reader
+    dtree = xd.io.open_cfradial1_datatree(cfradial1_file)
+
+    # Write to CfRadial2 NetCDF format
+    dtree.to_netcdf(temp_file)
```

Comment:
I'm not really sure, but you might have to add kwarg `engine="netcdf4"` here, since the tmp_file doesn't have an extension.